File: tarea2/5.java
import java.util.HashMap;
import java.util.Map;

public class CreditosMaterias {

    public static void main(String[] args) {
        HashMap<String, Integer> materias = new HashMap<>();
        materias.put("Matematicas", 4);
        materias.put("Programacion", 3);
        materias.put("Base de datos", 3);
        materias.put("Fisica", 4);
        materias.put("Quimica", 3);
        materias.put("Biologia", 3);
        materias.put("Ingles", 2);
        materias.put("Historia", 2);
        materias.put("Filosofia", 2);
        materias.put("Educacion fisica", 2);
        materias.put("Arte", 2);
        materias.put("Geografia", 2);

        for (Map.Entry<String, Integer> entry : materias.entrySet()) {
            System.out.println(entry.getKey() + " tiene " + entry.getValue() + " creditos");
        }

        int totalCreditos = 0;
        for (int creditos : materias.values()) {
            totalCreditos += creditos;
        }
        System.out.println("El total de creditos es: " + totalCreditos);
    }
}

File: tarea2/5.py
def creditosMaterias():
    materias={
        "Matematicas": 4,
        "Programacion": 3,
        "Base de datos": 3,
        "Fisica": 4,
        "Quimica": 3,
        "Biologia": 3,
        "Ingles": 2,
        "Historia": 2,
        "Filosofia": 2,
        "Educacion fisica": 2,
        "Arte": 2,
        "Geografia": 2}
    for materia in materias:
        print(f"{materia} tiene {materias[materia]} creditos")
    print("El total de creditos es: ", sum(materias.values()))
    
creditosMaterias()
